Cargo.toml

```diff
@@ -25,7 +25,7 @@ app_dirs = { version = "2", package = "app_dirs2" }
 clap = { version = "4", features = ["std", "derive", "help", "usage", "cargo", "error-context", "color", "wrap_help"], default-features = false }
 env_logger = { version = "0.11", optional = true }
 log = "0.4"
-reqwest = { version = "0.12.5", features = ["blocking"], default-features = false }
+reqwest = { version = "0.12.9", features = ["blocking"], default-features = false }
```

Comment:
Since the last time, I have found the following [ureq](https://crates.io/crates/ureq). It's a blocking HTTP request library which also supports all the TLS features we need and is probably not as heavy as reqwest. Since you are already caught up on the topic, what do you think about it?

Reply:
`ureq` Looks promising. I want to see the build size with `reqwest` difference first. The compiler might be too good to remove the unused code to see a difference. We will see from the data.

Reply:
Cool, see #415